### PR TITLE
Updated Podfile to the more fancy Ruby syntax

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Run the app again and click "Restore Default Status Bar". Resetting the iOS Simu
 Yes! SimulatorStatusMagic is also available through [CocoaPods](http://cocoapods.org). To install, simply add the following line to your Podfile:
 
 ```ruby
-pod 'SimulatorStatusMagic', :configurations => ['Debug']
+pod 'SimulatorStatusMagic', configurations: ['Debug']
```
Comment (kylef) [inline README.md]: In my opinion, I'd suggest to keep it consistent with all the guides on CocoaPods using the hash rocket. People who don't know Ruby may get confused with the differences, especially when trying to merge two different styles together while looking at the guides.

Example:

```ruby
pod 'SimulatorStatusMagic', configurations: ['Debug'], :git => 'https://github.com/shinydevelopment/SimulatorStatusMagic'
```

**Comment by KrauseFx** [comment]
Okay, thanks for your feedback @kylef. How did you see my pull request?

**Comment by kylef** [comment]
@KrauseFx I watch everything :grimacing:

**Comment by daveverwer** [comment]
I'm fine with either syntax but I agree with Kyle so I agree probably best to leave it. I think we did it this way copying it from the CocoaPods documentation actually.

**Pull request closed without merging**